cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -58,6 +58,10 @@ template <>
 struct corresponding_operator<aggregation::COUNT> {
   using type = DeviceCount;
 };
+template <>
+struct corresponding_operator<aggregation::MEAN> {
```

Comment:
This definitely doesn't look right to me. MEAN shouldn't have a corresponding operator.

Reply:
I was reluctant to add this or add one more function variant.